.github/workflows/testserver-helios.yml

```diff
@@ -0,0 +1,364 @@
+name: Deploy to Testserver with Helios
+
+on:
+  workflow_dispatch:
+    inputs:
+      HELIOS_TRIGGERED_BY:
+        description: "Username that triggered deployment"
+        required: true
+        type: string
+      HELIOS_BRANCH_NAME:
+        description: "Which branch to deploy"
+        required: true
+        type: string
+      HELIOS_BRANCH_HEAD_SHA:
+        description: "SHA of the branch to deploy"
+        required: true
+        type: string
+      HELIOS_PR_NUMBER:
+        description: "PR number that triggered deployment"
+        required: false
+        type: string
+      HELIOS_ENVIRONMENT_NAME:
+        description: "Which environment to deploy (e.g. artemis-test7.artemis.cit.tum.de, etc.)."
+        required: true
+        type: string
+      HELIOS_BUILD:
+        description: "Whether to also build or just deploy the existing Docker image"
+        required: true
+        type: boolean
+        default: false
+      HELIOS_BUILD_TAG:
+        description: "Docker tag to use if we are building or pulling an existing image"
+        required: false
+        type: string
+      HELIOS_RAW_URL:
+        description: "URL to the raw content of the repository in the format https://raw.githubusercontent.com/:owner/:repo/:sha"
+        required: false
+        type: string
```

Comment:
Why is this not marked as required? If this is not present the workflow would fail.